package.json

```diff
@@ -30,8 +30,8 @@
     "test:generate-schemas": "node ./test/scripts/generate-schemas.js"
   },
   "dependencies": {
-    "express-graphql": ">=0.12.0",
-    "graphql": "^15"
+    "graphql": "^16",
+    "graphql-http": "^1.16.0"
```

Comment:
What do we set the `graphql-http` version to?

Reply:
I. think `^1.16.0` is okay, but that always depends on the package.